### PR TITLE
Multinode for Ubuntu

### PR DESCRIPTION
**Pull request by technowhizz**
This PR enables support for Ubuntu Multinode Creation. Tested with Ubuntu 22.04

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -119,15 +119,17 @@ Generate Terraform variables:
...
    storage_flavor = "general.v1.small"
    storage_disk_size = 100
 
    EOF
 
-You will need to set the `multinode_flavor`, `multinode_keypair`, `prefix`, and
-`ssh_public_key`.
+You will need to set the `multinode_flavor`, `multinode_keypair`, `prefix`,
+`ssh_public_key`, and optionally `ssh_user` if you are using a different
+distribution.
```
Comment (Alex-Welsh) [inline README.rst:132]: 'a different distribution' might be confusing for someone using a multinode for the first time. Probably safest just to say something like 'a distribution that does not have a `cloud-user` account'

Comment (technowhizz) [reply README.rst:132]: Resolved

File: variables.tf [hunk 2]
```diff
@@ -6,6 +6,10 @@ variable "ssh_public_key" {
   type = string
 }
 
+variable "ssh_user" {
+  type = string
```
Comment (markgoddard) [inline variables.tf:10]: Docs say it's optional, but there is no default value.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 2 above)
Comment (Alex-Welsh) [reply variables.tf:10]: I agree it should have a default here as well, but it is mentioned in the readme inside the example tfvars file

Comment (markgoddard) [reply variables.tf:10]: > and optionally `ssh_user` if you are using a different distribution

Comment (markgoddard) [reply variables.tf:10]: Given that we don't have a default image, it probably doesn't make sense to have a default SSH user

Comment (technowhizz) [reply variables.tf:10]: Changed it in the docs

**Review by markgoddard — APPROVED** [review]

**Pull request merged**